app/models/publisher.rb

```diff
@@ -256,6 +256,10 @@ def register_for_2fa_removal
     )
   end
 
+  def is_registered_for_2fa_removal?
```

Comment:
I'm finding this logic kind of hard to read in the delete channels job because when the publisher is registered for 2fa it won't raise an error?

```ruby 
def not_registered_for_2fa_removal?
  two_factor_authentication_removal.blank?
end
```

Reply:
IMO the naming could be `registered_for_2fa_removal?` is preferable per rubocop style guide https://github.com/rubocop-hq/ruby-style-guide (I realize it's semi-arbitrary as Airbnb's Ruby style guide doesn't care but Rubocop does):

```
Avoid prefixing predicate methods with the auxiliary verbs such as is, does, or can. These words are redundant and inconsistent with the style of boolean methods in the Ruby core library, such as empty? and include?. [link]

# bad
class Person
  def is_tall?
    true
  end

  def can_play_basketball?
    false
  end

  def does_like_candy?
    true
  end
end

# good
class Person
  def tall?
    true
  end

  def basketball_player?
    false
  end

  def likes_candy?
    true
  end
end
```

Reply:
👍

Reply:
👍 will update to `registered_for_2fa_removal?`